.github/ISSUE_TEMPLATE/are-you-lost-.md

```diff
@@ -1,7 +1,6 @@
 ---
-name: Are you lost?
-about: If you did not find a bug and do not want to suggest a new feature, you probably
-  have some questions.
+name: Ask a quetion
```

Comment:
For asking questions we should point the users to https://community.privacyidea.org.